packages/headless/src/features/query-suggest/query-suggest-analytics-actions.ts

```diff
@@ -26,7 +26,9 @@ export const logQuerySuggestionClick = (
     }
   )();
 
-export type OmniboxSuggestionMetadata = OmniboxSuggestionsMetadata;
+export type OmniboxSuggestionMetadata = OmniboxSuggestionsMetadata & {
+  querySuggestResponseId: string;
```

Comment:
I will remove this line after bumping the `coveo.analytics` packaged once [this PR](https://github.com/coveo/coveo.analytics.js/pull/300) is merged.